Recent objc2's commit breaks the build.

Recent objc2's commit breaks the build on macOS v13.0.

Error messages are:

```
error: must specify the desired selector using `#[method(...)]` or `#[method_id(...)]`
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:26:1
   |
26 | / extern_methods!(
27 | |     unsafe impl NSView {
28 | |         pub(crate) fn window(&self) -> Option<Id<NSWindow, Shared>> {
29 | |             unsafe { msg_send_id![self, window] }
...  |
50 | |     }
51 | | );
   | |_^
   |
   = note: this error originates in the macro `$crate::__extract_custom_attributes` which comes from the expansion of the macro `extern_methods` (in Nightly builds, run with -Z macro-backtrace for more info)

error: must specify the desired selector using `#[method(...)]` or `#[method_id(...)]`
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/window.rs:24:1
   |
24 | / extern_methods!(
25 | |     unsafe impl NSWindow {
26 | |         #[sel(convertRectToScreen:)]
27 | |         pub(crate) fn convert_rect_to_screen(&self, rect: NSRect) -> NSRect;
...  |
31 | |     }
32 | | );
   | |_^
   |
   = note: this error originates in the macro `$crate::__extract_custom_attributes` which comes from the expansion of the macro `extern_methods` (in Nightly builds, run with -Z macro-backtrace for more info)

error: must specify the desired selector using `#[method(...)]` or `#[method_id(...)]`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:339:1
    |
339 | / declare_class!(
340 | |     pub(crate) struct PlatformNode {
341 | |         // SAFETY: This is set in `PlatformNode::new` immediately after
342 | |         // the object is created.
...   |
745 | |     }
746 | | );
    | |_^
    |
    = note: this error originates in the macro `$crate::__extract_custom_attributes` which comes from the expansion of the macro `declare_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/responder.rs:6:27
  |
6 | use objc2::{extern_class, foundation::NSObject, ClassType};
  |                           ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:8:5
  |
8 |     foundation::{CGFloat, NSObject, NSPoint, NSRect},
  |     ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/window.rs:8:5
  |
8 |     foundation::{NSObject, NSPoint, NSRect},
  |     ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/accessibility_constants.rs:6:12
  |
6 | use objc2::foundation::NSString;
  |            ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/accessibility_element.rs:6:27
  |
6 | use objc2::{extern_class, foundation::NSObject, ClassType};
  |                           ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/accessibility_functions.rs:6:12
  |
6 | use objc2::foundation::{NSObject, NSString};
  |            ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/context.rs:9:5
  |
9 |     foundation::MainThreadMarker,
  |     ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved imports `objc2::foundation`, `objc2::ns_string`
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:18:5
   |
18 |     foundation::{
   |     ^^^^^^^^^^ could not find `foundation` in `objc2`
...
21 |     msg_send_id, ns_string,
   |                  ^^^^^^^^^ no `ns_string` in the root

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/util.rs:8:12
  |
8 | use objc2::foundation::{NSPoint, NSRange, NSRect, NSSize};
  |            ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/adapter.rs:9:5
  |
9 |     foundation::{MainThreadMarker, NSArray, NSObject, NSPoint},
  |     ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
 --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/event.rs:8:12
  |
8 | use objc2::foundation::NSString;
  |            ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/subclass.rs:14:5
   |
14 |     foundation::{NSArray, NSObject, NSPoint},
   |     ^^^^^^^^^^ could not find `foundation` in `objc2`

error[E0432]: unresolved import `objc2::foundation`
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/lib.rs:22:16
   |
22 | pub use objc2::foundation::{NSArray, NSObject, NSPoint};
   |                ^^^^^^^^^^ could not find `foundation` in `objc2`

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:32:11
   |
32 |         #[sel(bounds)]
   |           ^^^

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:35:11
   |
35 |         #[sel(convertRect:toView:)]
   |           ^^^

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:38:11
   |
38 |         #[sel(convertPoint:fromView:)]
   |           ^^^

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:45:11
   |
45 |         #[sel(isFlipped)]
   |           ^^^

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:48:11
   |
48 |         #[sel(backingScaleFactor)]
   |           ^^^

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/window.rs:26:11
   |
26 |         #[sel(convertRectToScreen:)]
   |           ^^^

error: cannot find attribute `sel` in this scope
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/window.rs:29:11
   |
29 |         #[sel(convertPointFromScreen:)]
   |           ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:353:11
    |
353 |         #[sel(accessibilityParent)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:369:11
    |
369 |         #[sel(accessibilityChildren)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:374:11
    |
374 |         #[sel(accessibilityChildrenInNavigationOrder)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:380:11
    |
380 |         #[sel(accessibilityFrame)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:396:11
    |
396 |         #[sel(accessibilityRole)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:404:11
    |
404 |         #[sel(accessibilityTitle)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:417:11
    |
417 |         #[sel(accessibilityValue)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:436:11
    |
436 |         #[sel(setAccessibilityValue:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:442:11
    |
442 |         #[sel(accessibilityMinValue)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:452:11
    |
452 |         #[sel(accessibilityMaxValue)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:462:11
    |
462 |         #[sel(isAccessibilityElement)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:468:11
    |
468 |         #[sel(isAccessibilityFocused)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:474:11
    |
474 |         #[sel(setAccessibilityFocused:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:490:11
    |
490 |         #[sel(accessibilityPerformPress)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:502:11
    |
502 |         #[sel(accessibilityPerformIncrement)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:514:11
    |
514 |         #[sel(accessibilityPerformDecrement)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:526:11
    |
526 |         #[sel(accessibilityNotifiesWhenDestroyed)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:531:11
    |
531 |         #[sel(accessibilityNumberOfCharacters)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:543:11
    |
543 |         #[sel(accessibilitySelectedText)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:557:11
    |
557 |         #[sel(accessibilitySelectedTextRange)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:570:11
    |
570 |         #[sel(accessibilityInsertionPointLineNumber)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:583:11
    |
583 |         #[sel(accessibilityRangeForLine:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:596:11
    |
596 |         #[sel(accessibilityRangeForPosition:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:616:11
    |
616 |         #[sel(accessibilityStringForRange:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:630:11
    |
630 |         #[sel(accessibilityFrameForRange:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:655:11
    |
655 |         #[sel(accessibilityLineForIndex:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:668:11
    |
668 |         #[sel(accessibilityRangeForIndex:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:681:11
    |
681 |         #[sel(setAccessibilitySelectedTextRange:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot find attribute `sel` in this scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:692:11
    |
692 |         #[sel(isAccessibilitySelectorAllowed:)]
    |           ^^^
    |
note: `sel` is imported here, but it is a function-like macro
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:24:5
    |
24  |     sel, ClassType,
    |     ^^^

error: cannot determine resolution for the macro `ns_string`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:226:34
    |
226 |             Role::DocSubtitle => ns_string!("AXHeading"),
    |                                  ^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the macro `ns_string`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:151:34
    |
151 |             Role::RootWebArea => ns_string!("AXWebArea"),
    |                                  ^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the macro `ns_string`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:127:32
    |
127 |             Role::InputTime => ns_string!("AXTimeField"),
    |                                ^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the macro `ns_string`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:123:30
    |
123 |             Role::Heading => ns_string!("Heading"),
    |                              ^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the macro `ns_string`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:101:31
    |
101 |             Role::DateTime => ns_string!("AXDateField"),
    |                               ^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the macro `ns_string`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:100:27
    |
100 |             Role::Date => ns_string!("AXDateField"),
    |                           ^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the macro `ns_string`
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:46:33
   |
46 |             Role::ListMarker => ns_string!("AXListMarker"),
   |                                 ^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

   Compiling sha2 v0.10.6
error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/responder.rs:8:1
   |
8  | / extern_class!(
9  | |     #[derive(Debug)]
10 | |     pub struct NSResponder;
11 | |
...  |
14 | |     }
15 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `NSResponder`
   |   expected `&objc2::runtime::Object` because of return type
   |
   = note: expected reference `&objc2::runtime::Object`
              found reference `&NSResponder`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/responder.rs:8:1
   |
8  | / extern_class!(
9  | |     #[derive(Debug)]
10 | |     pub struct NSResponder;
11 | |
...  |
14 | |     }
15 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `NSResponder`
   |   expected `&mut objc2::runtime::Object` because of return type
   |
   = note: expected mutable reference `&mut objc2::runtime::Object`
              found mutable reference `&mut NSResponder`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:16:1
   |
16 | / extern_class!(
17 | |     #[derive(Debug)]
18 | |     pub(crate) struct NSView;
19 | |
...  |
23 | |     }
24 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `view::NSView`
   |   expected `&objc2::runtime::Object` because of return type
   |
   = note: expected reference `&objc2::runtime::Object`
              found reference `&view::NSView`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:16:1
   |
16 | / extern_class!(
17 | |     #[derive(Debug)]
18 | |     pub(crate) struct NSView;
19 | |
...  |
23 | |     }
24 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `view::NSView`
   |   expected `&mut objc2::runtime::Object` because of return type
   |
   = note: expected mutable reference `&mut objc2::runtime::Object`
              found mutable reference `&mut view::NSView`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/window.rs:14:1
   |
14 | / extern_class!(
15 | |     #[derive(Debug)]
16 | |     pub(crate) struct NSWindow;
17 | |
...  |
21 | |     }
22 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `NSWindow`
   |   expected `&objc2::runtime::Object` because of return type
   |
   = note: expected reference `&objc2::runtime::Object`
              found reference `&NSWindow`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/window.rs:14:1
   |
14 | / extern_class!(
15 | |     #[derive(Debug)]
16 | |     pub(crate) struct NSWindow;
17 | |
...  |
21 | |     }
22 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `NSWindow`
   |   expected `&mut objc2::runtime::Object` because of return type
   |
   = note: expected mutable reference `&mut objc2::runtime::Object`
              found mutable reference `&mut NSWindow`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/accessibility_element.rs:8:1
   |
8  | / extern_class!(
9  | |     #[derive(Debug)]
10 | |     pub struct NSAccessibilityElement;
11 | |
...  |
14 | |     }
15 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `accessibility_element::NSAccessibilityElement`
   |   expected `&objc2::runtime::Object` because of return type
   |
   = note: expected reference `&objc2::runtime::Object`
              found reference `&accessibility_element::NSAccessibilityElement`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/accessibility_element.rs:8:1
   |
8  | / extern_class!(
9  | |     #[derive(Debug)]
10 | |     pub struct NSAccessibilityElement;
11 | |
...  |
14 | |     }
15 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `accessibility_element::NSAccessibilityElement`
   |   expected `&mut objc2::runtime::Object` because of return type
   |
   = note: expected mutable reference `&mut objc2::runtime::Object`
              found mutable reference `&mut accessibility_element::NSAccessibilityElement`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `extern_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:339:1
    |
339 | / declare_class!(
340 | |     pub(crate) struct PlatformNode {
341 | |         // SAFETY: This is set in `PlatformNode::new` immediately after
342 | |         // the object is created.
...   |
745 | |     }
746 | | );
    | | ^
    | | |
    | |_expected struct `objc2::runtime::Object`, found struct `PlatformNode`
    |   expected `&objc2::runtime::Object` because of return type
    |
    = note: expected reference `&objc2::runtime::Object`
               found reference `&PlatformNode`
    = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `declare_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/node.rs:339:1
    |
339 | / declare_class!(
340 | |     pub(crate) struct PlatformNode {
341 | |         // SAFETY: This is set in `PlatformNode::new` immediately after
342 | |         // the object is created.
...   |
745 | |     }
746 | | );
    | | ^
    | | |
    | |_expected struct `objc2::runtime::Object`, found struct `PlatformNode`
    |   expected `&mut objc2::runtime::Object` because of return type
    |
    = note: expected mutable reference `&mut objc2::runtime::Object`
               found mutable reference `&mut PlatformNode`
    = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `declare_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/subclass.rs:38:1
   |
38 | / declare_class!(
39 | |     struct AssociatedObject {
40 | |         // SAFETY: These are set in AssociatedObject::new, immediately after
41 | |         // the object is created.
...  |
49 | |     }
50 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `AssociatedObject`
   |   expected `&objc2::runtime::Object` because of return type
   |
   = note: expected reference `&objc2::runtime::Object`
              found reference `&AssociatedObject`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `declare_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
  --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/subclass.rs:38:1
   |
38 | / declare_class!(
39 | |     struct AssociatedObject {
40 | |         // SAFETY: These are set in AssociatedObject::new, immediately after
41 | |         // the object is created.
...  |
49 | |     }
50 | | );
   | | ^
   | | |
   | |_expected struct `objc2::runtime::Object`, found struct `AssociatedObject`
   |   expected `&mut objc2::runtime::Object` because of return type
   |
   = note: expected mutable reference `&mut objc2::runtime::Object`
              found mutable reference `&mut AssociatedObject`
   = note: this error originates in the macro `$crate::__inner_extern_class` which comes from the expansion of the macro `declare_class` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0599]: no method named `class` found for struct `view::NSView` in the current scope
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/subclass.rs:128:46
    |
128 |           let prev_class = unsafe { &*((*view).class() as *const Class) };
    |                                                ^^^^^ this is an associated function, not a method
    |
   ::: /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/accesskit_macos-0.3.0/src/appkit/view.rs:16:1
    |
16  | / extern_class!(
17  | |     #[derive(Debug)]
18  | |     pub(crate) struct NSView;
19  | |
...   |
23  | |     }
24  | | );
    | |_- method `class` not found for this struct
    |
    = note: found the following associated functions; to be used as methods, functions must have a `self` parameter
note: the candidate is defined in the trait `ClassType`
   --> /Users/allen/.cargo/registry/src/github.com-1ecc6299db9ec823/objc2-0.3.0-beta.4/src/class_type.rs:85:5
    |
85  |     fn class() -> &'static Class;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: use associated function syntax instead
    |
128 |         let prev_class = unsafe { &*(view::NSView::class() as *const Class) };
    |                                      ~~~~~~~~~~~~~~~~~~~
help: disambiguate the associated function for the candidate
    |
128 |         let prev_class = unsafe { &*(<view::NSView as ClassType>::class((*view)) as *const Class) };
    |                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/platforms/macos/Cargo.toml
+++ b/platforms/macos/Cargo.toml
@@ -13,6 +13,6 @@ edition = "2021"
 [dependencies]
 accesskit = { version = "0.8.1", path = "../../common" }
 accesskit_consumer = { version = "0.11.0", path = "../../consumer" }
-objc2 = "0.3.0-beta.3"
+objc2 = "=0.3.0-beta.3"
 once_cell = "1.13.0"
 parking_lot = "0.12.1"

